### ISSUE TITLE
Do not complain about ssh errors of offline nodes

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#1187]
Originally reported of Google Code with ID 1131.

```
After running `gnt-cluster renew-crypto --new-ssh-keys` successfully, I'm getting SSH errors about a node which is offline. 
```
Originally added on 2015-10-13 12:38:21 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Fixed in 2.13 with:

commit 556b339c987c0686ce2f7edbf5012c18afeb8af3
Author: Helga Velroyen <helgav@google.com>
Date:   Wed Oct 14 10:24:33 2015 +0200

    Don't warn about broken SSH setup of offline nodes
    
    This fixes issue 1131. 'gnt-cluster verify' should stop
    complaining about broken SSH setups of offline nodes.
    
    Additionally, this fixes a problem when readding nodes.
    In some cases, Ganeti complains about a possible attack,
    which is a valid case for readding a node (if a key
    renew took place between offlining and readding the node).
    
    Signed-off-by: Helga Velroyen <helgav@google.com>
    Reviewed-by: Klaus Aehlig <aehlig@google.com>

```
Originally added on 2015-10-14 11:46:12 +0000 UTC.

Mentioned Issue with GoogleCode ID 1131 has been migrated to GitHub with ID #1187.
Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2016-01-11 15:04:18 +0000 UTC.

Changed State: Released
